mailet/icalendar/src/main/resources/ical4j.properties

```diff
@@ -0,0 +1,5 @@
+ical4j.unfolding.relaxed=true
+ical4j.parsing.relaxed=true
+ical4j.validation.relaxed=true
+ical4j.compatibility.outlook=true
+ical4j.compatibility.notes=true
```

Comment:
This can be done interacting with a static configuration object in code.

I find this cleaner...